xarray/backends/api.py

```diff
@@ -80,6 +81,30 @@
 }
 
 
+def avail_engines() -> pd.DataFrame:
```

Comment:
Shall we instead always return `str` by using `repr(eng_df)` and `eng_df._repr_html()` depending of `OPTIONS["display_style"]`. This would make it easy to copy paste into a bug report

Reply:
I didn't make this change because it feels like it's part of the biggest unresolved bit here. I'll note I am planning on joining in on Wednesday's dev meeting this week.